File: S5/HLIN505 - Modélisation et Programmation par Objets 2/TP/2016-1017/TP3/Pokemon/src/PokemonSportif.java
public class PokemonSportif extends PokemonPhysique {
	private int frequenceCardiaque;
	public PokemonSportif(String name , double poids , int nombreDePattes , double taille , int frequenceCardiaque) {
		super(name , poids , nombreDePattes , taille);
		this.frequenceCardiaque = frequenceCardiaque;
	}
	public String toString() {
		return super.toString() + " et ma fréquence cardiaque est de " + frequenceCardiaque + " pulsations par minutes";
	}
}

File: S6/HLIN603 - Objets avances/TPs/2016-2017/TP1/Exercice 2/source/PokemonSportif.cc
#include "../header/PokemonSportif.h"
#include <sstream>

PokemonSportif::PokemonSportif(std::string s , int n , int m , float t , int b)
: PokemonTerrestre(s , n , m , t) , battementsParMinute(b) {}

std::string PokemonSportif::toString() {

  std::stringstream s;
  std::string sBattementsParMinute;
  s << this->battementsParMinute;
  s >> sBattementsParMinute;

  return PokemonTerrestre::toString() + " et ma fréquence cardiaque est de " + sBattementsParMinute + " battements par minute";
}
